MLS.Agent/Program.cs

```diff
@@ -27,11 +26,11 @@ namespace MLS.Agent
 {
     public class Program
     {
-        private static readonly ServiceCollection _serviceCollection = new ServiceCollection();
+        private static readonly ServiceCollection ServiceCollection = new ServiceCollection();
```

Comment:
The previous name is the normal format for fields. Hungarian-ish, but we should be consistent.

Reply:
static and const tend to be with that naming (it might be something inherited from resharper settings)

Reply:
I think it's only when not `private`.